addons/hr_holidays/models/hr_holidays.py

```diff
@@ -375,6 +383,10 @@ def unlink(self):
             raise UserError(_('You cannot delete a leave which is in %s state.') % (holiday.state,))
         return super(Holidays, self).unlink()
 
+    @api.multi
+    def copy_data(self, default=None):
```

Comment:
Why ?

Reply:
Was a request of the HR, else people who have rights duplicate their approved holidays and then change the date...